lib/cli.js

```diff
@@ -338,9 +339,7 @@ module.exports = async (argv) => {
 	let formatter = cli.flags.formatter;
 
 	if (cli.flags.customFormatter) {
-		const customFormatter = path.isAbsolute(cli.flags.customFormatter)
-			? cli.flags.customFormatter
-			: path.join(process.cwd(), cli.flags.customFormatter);
+		const customFormatter = resolveCustomFormatter(cli.flags.customFormatter);
 
```

Comment:
Could you add a test case to cover the code calling `resolveCustomFormatter()` to `cli.test.js`?